lib/glific/templates.ex

```diff
@@ -211,7 +211,10 @@ defmodule Glific.Templates do
   end
 
   defp contains_emoji?(text) do
```

Comment:
can you add spec for this function and `contains_invalid_chars?` and all the functions inside it?